docker/_include.sh

```diff
@@ -1,6 +1,6 @@
 DOCKER_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-if [ `uname` == 'Linux' ]; then
+if [ `uname` == 'Linux' -o `uname` == 'Darwin' ]; then
```

Comment:
funny, I'm on OS X too and this worked for me. Feel free to revert. It seems the docker automation needs some Mac love.